app/src/main/java/org/eyeseetea/malariacare/views/question/CommonQuestionView.java

```diff
@@ -0,0 +1,34 @@
+package org.eyeseetea.malariacare.views.question;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.widget.LinearLayout;
+
+public class CommonQuestionView extends LinearLayout implements IValidationQuestion {
+    boolean isActive;
+
+    public CommonQuestionView(Context context) {
+        super(context);
+    }
+
+    public CommonQuestionView(Context context,
+            @Nullable AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public CommonQuestionView(Context context,
+            @Nullable AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @Override
+    public boolean isActive() {
```

Comment:
default value for isActive is false, this is dangerous because only is setting when is a child question. I think default value to true is better